index.html

```diff
@@ -1375,6 +1376,10 @@ <h3>
           <li>Await the result of performing a <a>fetch</a> with
           <var>request</var>, letting <var>response</var> be the result.
           </li>
+          <li>If [=header list/extract a MIME type=] from |response|'s
+          [=response/header list=] is failure or not a [=JSON MIME Type=], then
+          abort these steps.
```

Comment:
This seems out of step with the earlier text, which "strongly encourages" ... "at least a" JSON MIME type. If we're going to abort the fetching of a manifest for a non-JSON MIME-type, should the earlier text be more explicit?

Reply:
Also, do we know if this is going to break sites? I don't think implementations currently check that the MIME type is JSON. There are probably a lot of sites that don't serve their manifest with the appropriate MIME type which will break if we add this behaviour.

Reply:
I can confirm Chromium doesn't check the MIME type of the response - we just decode it as UTF8 and try parsing it and see if that succeeds or not.

Reply:
Agree, and I'm also worried about potential breakage, although I don't know how bad it will be. 

We might need to reach out to someone to run a query for us on HTTP Archive or big table? Or see if there is some other way of getting this data. (e.g., via the https://github.com/webdevdata tools)